fenix/app/src/main/java/org/mozilla/fenix/HomeActivity.kt

```diff
@@ -523,6 +526,14 @@ open class HomeActivity : LocaleAwareAppCompatActivity(), NavHostActivity {
         BrowsersCache.resetAll()
     }
 
+    override fun onProvideAssistContent(outContent: AssistContent?) {
+        super.onProvideAssistContent(outContent)
+        if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.M) {
```

Comment:
The documentation mentions "On Pixel devices running Android 12 or higher" so I'm curious why are we using a lower api level check here?
Based on the fact that this method [was added in API 23](https://developer.android.com/reference/android/app/Activity#onProvideAssistContent(android.app.assist.AssistContent)) and the official examples[1][2] not doing an api check maybe we can avoid it also?



[1] https://developer.android.com/guide/components/activities/recents#url-sharing
[2] https://developer.android.com/training/articles/assistant.html#source_app

Reply:
Yes you're right! Thanks for sharing the assistant link as well, TIL! 👌 I updated to remove the if check and added the annotation to suppress the lint issue as `setWebUrl` function along the function was added in api 23.